src/components/style/FilterSizes.css

```diff
@@ -0,0 +1,55 @@
+.more-filters-container {
+    position: relative;
+    z-index: auto;
+    display: flex;
+    flex-flow: row wrap;
+}
+
+#size-img {
+    width: 100%;
+    height: auto;
+}
+
+.button {
+    content: "";
+    width: 2vw;
+    height: 2vw;
+    border-radius: 100%;
+    background-color: #EBF9FE;
+    border: #1EBDF4 2px solid;
+    box-shadow: rgba(0, 0, 0, 0.2) 1px 1px 8px;
+    position: absolute;
+    z-index: 1;
+    translate: -50%, -50%;
+    
+}
+
+.button:hover {
+    background-color: #1EBDF4;
+}
+
+.clicked {
+    background-color: #1EBDF4;
+}
+
+.speech-bubble {
+    background-color: #EBF9FE;
+    border: #1EBDF4 2px solid;
+    border-radius: 15px;
+    padding: 1em;
+    display: flex;
+    flex-direction: column;
+    width: stretch;
+    margin-top: 0.5em;
+    min-height: 2.5em;
+    /* animation: fadeIn 2s ease; */
```

Comment:
Does this help with the fading-in and size growing? AFAIK adding transition/animation should make the element "re-render" the animation every time the internal component changes, but I could be wrong here! 

(if you want a hack-y solution, you can always toggle the class on-and-off: it should transition in and out)